### ISSUE TITLE
CLI -  Inspect command is broken

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#17889]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- use rancher CLI v2.2.0-rc5 to access to a rancher server v2.0.11
- use the inspect command to get info about cluster and projects 

**Result:**
```
➜  ~ rancher -v
rancher version v2.2.0-rc5
➜  ~ rancher node
ID                NAME                   STATE     POOL                  DESCRIPTION
c-44m8t:m-9s77g   jiaqi-do-all-roles-2   active    jiaqi-do-all-roles-
c-44m8t:m-dd7gf   jiaqi-do-all-roles-3   active    jiaqi-do-all-roles-
c-44m8t:m-hqhcp   jiaqi-do-all-roles-1   active    jiaqi-do-all-roles-
➜  ~ rancher inspect  --type node c-44m8t:m-9s77g
FATA[0001] unknown resource type
➜  ~ rancher inspect c-44m8t:m-9s77g
FATA[0001] unknown resource type
➜  ~ rancher projects
ID                NAME         STATE     DESCRIPTION
c-44m8t:p-5bzl4   System       active    System project created for the cluster
c-44m8t:p-gwfjs   Default      active    Default project created for the cluster
c-44m8t:p-ln9wn   mystep1-p1   active
➜  ~ rancher inspect c-44m8t:p-ln9wn
FATA[0001] unknown resource type
➜  ~ rancher inspect --type project c-44m8t:p-ln9wn
FATA[0001] unknown resource type
```
**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.0.11
- Installation option (single install/HA):
Single install

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
- Kubernetes version (use `kubectl version`): 

```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.5", GitCommit:"32ac1c9073b132b8ba18aa830f46b77dcceb0723", GitTreeState:"clean", BuildDate:"2018-06-21T11:46:00Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.6", GitCommit:"b1d75deca493a24a2f87eb1efde1a569e52fc8d9", GitTreeState:"clean", BuildDate:"2018-12-16T04:30:10Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
 17.3.2
```

**Comment by dramich** [comment]
Available in v2.2.0-rc6

**Comment by jiaqiluo** [comment]
The bug is reproduced in v2.2.0-rc5

The bug fix is validated in v2.2.0-rc6
